src/EventQuery.php

```diff
@@ -137,9 +137,11 @@ protected function init_end_date(array $params, DateTimeImmutable $start) : Date
    * Get the results for this EventQuery as a collection of zero or more Events
    *
    * @internal
+   * @return array
```

Comment:
@acobster Just put `list<\Timber\Post>` here.

Reply:
it means `array<int, \Timber\Post>`

Reply:
Good to know, but `array<int, \Timber\Post>` is not quite correct, is it? Wouldn't `array<\Timber\Post>` be better?

Reply:
Oh yes.